### DISCUSSION TITLE
Using Realtime Listener in React

### DISCUSSION THREAD

**Discussion by kaili-kameoka** [supabase/supabase#4343] [Questions] [answered]
I'm having trouble using on().subscribe() in my React Application. I've copied the example out of the documentation. The callback in the on() function doesn't seem to be running at all.

Here is a repo to reproduce the issue.https://github.com/kaili-kameoka/supabase-sandbox

**Comment by w3b6x9** [comment, marked as answer]
@kaili-kameoka judging from that repo's package-lock.json, looks like it's on realtime-js v1.3.2 while supabase-js is on v1.28.2. There was an issue with this. Please update to the latest supabase-js client version (1.28.5) and it should work.

**Reply by w3b6x9** [reply]
@kaili-kameoka feel free to reach out if the above does not fix your issue
